contracts/Market.sol

```diff
@@ -80,4 +101,16 @@ contract Market is UseState {
 		issuedMetrics += 1;
 		return address(metrics);
 	}
+
+	function getTotalVotes() public view returns (uint256) {
+		return totalVotes;
+	}
+
+	function setDEVtokenAddress(address _devAddress) public {
+		DEVaddress = _devAddress;
+	}
+
+	function activateMarket() public {
```

Comment:
Is not this method dangerous?

If it's necessary for testing, it might be a good idea to instantiate with the second argument set to `true`, or create a wrapped contract for testing.

case of instantiation:
```ts
const market = await marketContract.new('0x...', true)
```
case of wrapped contract:
```sol
contract MarketTest is Market {
    function activateMarket() public {
        // ...
    }
}
```